boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160.yaml

```diff
@@ -19,3 +19,4 @@ supported:
   - spi
   - watchdog
   - counter
+  - netif
```

Comment:
I'm not sure if we want to add the `netif` capability to the secure board definition. In practice, we only build networking for non-secure part.

Reply:
True, but do we really want/need to run tests in the secure domain?

Reply:
Done